test/HealthChecks.UI.Tests/HealthChecks.UI.approved.txt

```diff
@@ -24,6 +24,7 @@ namespace HealthChecks.UI.Configuration
     {
         public Settings() { }
         public HealthChecks.UI.Configuration.Settings AddHealthCheckEndpoint(string name, string uri) { }
+        public HealthChecks.UI.Configuration.Settings SetNotifyUnHealthyOneTimeUntilChange() { }
```

Comment:
Please move after SetMinimumSecondsBetweenFailureNotifications

Reply:
Thanks, it's been moved to be sorted alphabetically.